Packages/CommonUI/CommonUI/Classes/View/TextField/PasscodeTextField.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  SwiftUIView.swift
+//  
+//
+//  Created by đông on 17/03/2022.
+//
+
+import SwiftUI
+
+private enum Constants {
+	static let textFieldHeight = 52.0
+	static let paddingVertical = 14.0
+	static let paddingHorizontal = 24.0
+	static let spacing = 14.0
+	static let widthPasscode = 56.0
+}
+
+public struct PasscodeTextField: View {
+	// MARK: - Variables
+	@Binding var text: String
+	@Binding var inputStyle: TextInputStyle
+	private let keyboardType: UIKeyboardType
+
+	// MARK: - Init
+	public init(text: Binding<String>,
+				inputStyle: Binding<TextInputStyle>,
+				keyboardType: UIKeyboardType = .numberPad) {
+		self._text = text
+		self._inputStyle = inputStyle
+		self.keyboardType = keyboardType
+	}
+	public var body: some View {
+		TextField("", text: $text)
+			.font(font)
+			.foregroundColor(textColor)
+			.frame(width: Constants.widthPasscode, height: Constants.widthPasscode)
+			.background(backgroundColor)
+			.cornerRadius(cornerRadiusPasscode)
+	}
+}
+
+// MARK: - Private func
+private extension PasscodeTextField {
+	var backgroundColor: Color {
+		inputStyle.backgroundColor
+	}
+
+	var tintColor: Color {
+		inputStyle.tintColor
+	}
+
+	var borderColor: Color {
+		inputStyle.borderColor
+	}
+
+	var borderWidth: CGFloat {
+		inputStyle.borderWidth
+	}
+
+	var cornerRadius: CGFloat {
+		inputStyle.cornerRadius
+	}
+
+	var cornerRadiusPasscode: CGFloat {
+		inputStyle.cornerRadiusPasscode
+	}
+
+	var textColor: Color {
+		inputStyle.textColor
+	}
+
+	var notifyColor: Color {
+		inputStyle.notifyColor
+	}
+
+	var font: Font {
+		inputStyle.textStyle.font
+	}
+}
+
+struct PasscodeTextField_Previews: PreviewProvider {
+    static var previews: some View {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/4cd8df00e7c125732af8eec5bb3132605d130482/Packages/CommonUI/CommonUI/Classes/View/TextField/PasscodeTextField.swift#L82"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>PasscodeTextField.swift:82</code></td>
    </tr>
  </tbody>
</table>